common/src/main/java/com/redhat/cloud/notifications/models/NotificationHistory.java

```diff
@@ -32,17 +31,7 @@ public class NotificationHistory extends CreationTimestamped {
             "created", "nh.created",
             "invocation_time", "nh.invocationTime",
             "invocation_result", "nh.invocationResult",
-            "status", "nh.status",
-            // NOTIF-674 Delete after the frontend has been updated
-            "nh.created", "nh.created",
-            "invocationtime", "nh.invocationTime",
-            "invocationresult", "nh.invocationResult",
-            "endpoint_id", "endpoint_id"
```

Comment:
Is it normal that we're not allowing a sort on the endpoint anymore?

Reply:
> Is it normal that we're not allowing a sort on the endpoint anymore?

The only reason I can see is that sorting by `endpoint_id` doesn't make lots of sense to me :thinking: .

But the format is good and we could include it on the supported sorting. 
What do you think?

Reply:
I just checked the code. It doesn't make sense to sort on the `endpoint_id` indeed.